m2c/flow_graph.py

```diff
@@ -862,7 +860,7 @@ def find_block_by_label(label: str) -> Optional[Block]:
                             and isinstance(arg.argument, AsmGlobalSymbol)
                             and any(
                                 arg.argument.symbol_name.startswith(prefix)
-                                for prefix in ("jtbl", "jpt_", "lbl_")
+                                for prefix in ("jtbl", "jpt_", "lbl_", "_jumptable")
```

Comment:
just double checking: `_jumptable` vs `jumptable_`?

Reply:
You are right.